test/testMOI.jl

```diff
@@ -88,40 +13,29 @@ function runtests()
             end
         end
     end
+    return
 end
 
-"""
-    test_runtests()
-
-This function runs all the tests in MathOptInterface.Test.
-
-Pass arguments to `exclude` to skip tests for functionality that is not
-implemented or that your solver doesn't support.
-"""
 function test_runtests()
-    MOI.Test.runtests(
-        BRIDGED,
-        CONFIG,
-        exclude=[
-            "test_attribute_NumberOfThreads",
-            "test_quadratic_",
```

Comment:
This, for example, skipped all quadratic tests, so I don't think it's what you intended :smile: